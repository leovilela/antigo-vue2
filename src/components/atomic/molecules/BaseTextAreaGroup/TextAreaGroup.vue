<template>
  <div>
    <label for="" v-if="label">{{label}}</label>
    <Base-Textarea :value="value" @input="$emit('input', $event.target.value)" rows="10" cols="50" />
  </div>
</template>

<script>
export default {
  props: {
    value: {
      type: String,
      default: "",
    },
    label: {
      type: String,
      default: "",
    },
  },
};
</script>

<style>
</style>
