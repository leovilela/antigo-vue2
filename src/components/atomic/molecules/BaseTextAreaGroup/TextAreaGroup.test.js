import { shallowMount } from "@vue/test-utils";
import TextAreaGroup from "../BaseTextAreaGroup/TextAreaGroup";

const factory = () => shallowMount(TextAreaGroup, {
  stubs: ["Base-Textarea"],
});

describe("TextAreaGroup", () => {
  test("mounts properly", () => {
    const wrapper = factory();
    expect(wrapper.isVueInstance()).toBeTruthy();
  });

  test("renders properly", () => {
    const wrapper = factory();
    expect(wrapper.html()).toMatchSnapshot();
  });
});
