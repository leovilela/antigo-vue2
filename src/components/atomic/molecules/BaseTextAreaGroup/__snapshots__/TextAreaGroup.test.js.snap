// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextAreaGroup renders properly 1`] = `
<div>
  <!---->
  <base-textarea-stub value="" rows="10" cols="50"></base-textarea-stub>
</div>
`;
